src/cli.rs

```diff
@@ -12,6 +12,14 @@ pub(crate) enum Subcommand {
     /// Pull the image in the configuration file
     #[structopt(name = "pull")]
     Pull {},
+
+    /// Generate shell completions to stdout.
+    #[structopt(name = "completion")]
+    Completion {
+        /// The shell to generate completions for.  Choose from: bash, fish, zsh, powershell, elvish
+        #[structopt(name = "SHELL", parse(try_from_str))]
+        shell: clap::Shell,
```

Comment:
AIUI `structopt` re-exports `clap`, so can we just use `structopt::clap::Shell` here without having to explicitly include it in Cargo.toml?

Reply:
cool.  Good dependency hygiene.